### ISSUE TITLE
does the touch controller work for you on the Core2?

### ISSUE THREAD

**Issue by mhaberler** [usedbytes/m5core2-basic-idf#3]
see https://github.com/lvgl/lvgl_esp32_drivers/issues/222#issuecomment-1637882391

**Comment by usedbytes** [comment]
Yes, works for me. The calibration isn't perfect - seems better towards the centre of the display (the LED switch is difficult to press, but the other two work well)

**Comment by mhaberler** [comment]
thanks - I must have bungled it up then
pity no JTAG with this device

any plans for the M5Stack CoreS3? it is quite a safari to debug the S3's reprodocibly, but I have a [working example](https://github.com/mhaberler/flowsensor) where it's sort of ok

**Comment by usedbytes** [comment]
No plans no, I haven't used the Core 2 at all since I got it, so can't see me using a CoreS3 either :-)
